app/controllers/api/v1/subscriptions_controller.rb

```diff
@@ -38,15 +34,58 @@ def terminate
             ),
           )
         else
-          validation_errors(result)
+          render_error_response(result)
         end
       end
 
+      def update
+        service = Subscriptions::UpdateService.new
+
+        result = service.update(**{
+          id: params[:id],
```

Comment:
To prevent malicious users to update subscriptions not attached to their organization, we should pass the `organization_id` as well

Reply:
Fixed!